src/index.js

```diff
@@ -1,13 +1,27 @@
 'use strict';
 import api from './api';
 import request from './request';
+import { getResults, getTrendingResults,
+          constructObj, constructTrendingObj } from './utilities';
+
+const interestHandler = {
+  processor: getResults,
+  objectConstructor: constructObj,
```

Comment:
Now that there are two of these, can you rename this one to `constructInterestObj` instead of `constructObj`

Reply:
And rename `getResults` to `getInterestResults`

Reply:
Sorry to be a pain 😛

Reply:
hey no problem... make sense here. I'll do so.